contracts/extbridge/ext_bridge.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity ^0.4.24;
+
+import "../bridge/Bridge.sol";
+import "./callback.sol";
+
+contract ExtBridge is Bridge {
+    address public callback = address(0);
+
+    constructor (bool _modeMintBurn) Bridge(_modeMintBurn) public payable {
+    }
+
+    function setCallback(address _addr) public {
+        callback = _addr;
+    }
+
+    // handleERC20Transfer sends the ERC20 token by the request and processes extended feature.
+    function handleERC20Transfer(
+        uint256 _amount,
+        address _to,
+        address _contractAddress,
+        uint64 _requestNonce,
+        uint64 _requestBlockNumber//,
```

Comment:
The comment seems unnecessary.

Reply:
I will uncomment in next PR. So I wrote it. :)